qa/tasks/iscsi_gwcli.py

```diff
@@ -0,0 +1,112 @@
+"""
```

Comment:
That's looks cool and will do it.

Reply:
@dillaman 
BTW, for the cram, such as for the following case:

```
# cat gwcli.t
  $ IP=`hostname -i | awk '{print $1}'`
  > HOST=`hostname`
  > gwcli iscsi-targets/iqn.2003-01.com.redhat.iscsi-gw:ceph-gw/gateways create ip_addresses=$IP gateway_name=$HOST
  $ gwcli ls iscsi-targets/iqn.2003-01.com.redhat.iscsi-gw:ceph-gw/gateways
    o- gateways .................................................................................................. [Up: 1/1, Portals: 1]
      o- lxbceph1 ................................................................................................. [192.168.0.174 (UP)]
```

Do you have any idea how to check the `gwcli ls` result ? Because the `gwcli ls` output will replace the `$IP` and `$HOST` as above.

Reply:
There will be some others' output like this.

Reply:
I ignored the detail IP checks.